### PR TITLE
Updates to conform to new ng doc example home

### LINKED ISSUE #25: Adjust to new example folder layout
See
- https://github.com/dart-lang/site-webdev/issues/464
- https://github.com/dart-lang/site-webdev/issues/468

cc @kwalrath

### PR DESCRIPTION
**Pull request by chalin**
Fixes #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,22 +8,20 @@ to individual repos under [angular-examples](https://github.com/angular-examples
...
-Run `sync_all` to efficiently synchronize all configured examples.
-
+The `--match` option actually takes a regular expression as an argument.
+To sync all examples you can use `.` as a "match-all" pattern:
 ```
-dart sync_all
+dart dart_doc_syncer --match .
 ```
 
 How to get the tools
```
Comment (kwalrath) [inline README.md]: Are there still multiple tools?

Comment (chalin) [reply README.md]: No. Thanks for catching that.

**Comment by kwalrath** [comment]
LGTM, but of course an engineer (@thso, I hope) should check the code.

**Comment by thso** [comment]
Nice change and simplifications!

**Pull request merged**